common/source/docs/common-loading-firmware-onto-pixhawk.rst

```diff
@@ -51,6 +49,14 @@ Mission Planner will try to detect which board you are using. It may ask you to
 
    Mission Planner: Install FirmwarePrompt
 
+Often you will be presented with a dropdown box of firmware variants for the board, which you can select from (such as bi-directional DShot variants, if available). For boards which share the Pixhawk board id, the list will be extensive, as shown below:
+
+.. image:: ../../../images/pixhawk-firmware.png
+   :target: ../_images/pixhawk-firmware.png
+
+Select the appropriate firmware for your board. For boards marked "Pixhawk", Pixhawk1 firmware is usually the best choice.
+
+.. warning:: Pixhawk1 boards supplied by non ArduPilot partners (ie. "clones") may not be fully compatible with ArduPilot and loading ArduPilot firmware may result in operational issues.
```

Comment:
maybe "some boards labelled as Pixhawk 2.4.x may have sensor substitutions which could cause issues. Please see the BARO_OPTIONS parameter for a workaround for a known substitution on some boards of a MS5607 barometer where a MS5611 should be used. IMUs may also be substituted. Where possible please source boards from reputable vendors"

Reply:
I'd suggest "ArduPilot Parnters" instead of "reputable vendors" maybe.